civis/io/_tables.py

```diff
@@ -32,11 +39,41 @@
     '|': 'pipe',
 }
 
+SQL_PANDAS_MAP = {
+    'smallint': np.int16,
+    'int': np.int32,
+    'integer': np.int32,
+    'bigint': np.int64,
+    'decimal': np.float64,
+    'numeric': np.float64,
+    'float': np.float64,
+    'float4': np.float32,
+    'float8': np.float64,
+    'real': np.float32,
+    'double precision': np.float64,
+    'boolean': np.bool,
+    'bool': np.bool,
```

Comment:
Doesn't redshift output 't' and 'f' for boolean values? http://docs.aws.amazon.com/redshift/latest/dg/r_Boolean_type.html

I think that means this would fail. I thought @stephen-hoover ran into this in CivisML?

Reply:
Yeah this won't work.

```python
In [1]: np.bool('t')
Out[1]: True

In [2]: np.bool('f')
Out[2]: True
```